### PR TITLE
Finished Project.

### PR DESCRIPTION
**Pull request by cod3smith**
This PR contains the full Apple Homepage as shown on the Odin Project.

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# add the linters you want stickler to use for this project
```
Comment (lucasmazo32) [inline .stickler.yml:1]: - [ ] Kindly fix the lintern errors.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                <li class="nav-item">
+                    <img src="./assets/images/apple-brands.svg" alt="Apple Icon">
+                </li>
+                <li class="nav-item">Store</li>
+                <li class="nav-item">Mac</li>
+                <li class="nav-item">iPod</li>
+                <li class="nav-item">iPhone</li>
+                <li class="nav-item">iPad</li>
+                <li class="nav-item">iTunes</li>
+                <li class="nav-item">Support</li>
```
Comment (lucasmazo32) [inline index.html:23]: - [ ] These items should be links. Kindly fix that.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    
+            </p>
+        </section>
+    </header>
+    
+    <div class="promo-cards">
+        <img src="./assets/images/promo_iphone5s.jpg" alt="iphone5s">
+        <img src="./assets/images/promo_iphone5c.jpg" alt="iphone5c">
+        <img src="./assets/images/promo_30_years.jpg" alt="Promo">
+        <img src="./assets/images/promo_supplier_responsibility.jpg" alt="Promo Responsibility.">
```
Comment (lucasmazo32) [inline index.html:47]: - [ ] These images should be links. Kindly fix that.

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by KelynPNjeri** [comment]
I implemented all the changes requested in the project. I hope it meets the rubric.

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @KelynPNjeri awesome job on this project :tada: :rocket: 
You did great on this job, also implementing the cahnges required by the previousTSE.
There are still just some tiny issues to fix before this project can be approved, follow the comments bellow :point_down: 
### :stop_sign: Required changes :construction: :construction_worker: 
- [x] The search input can be improved so it looks better, see the inline comment please
- [x] The promo-cards have some space gap between them, change the width to 100% to fix the issue, see the inline comment.
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :stop_sign: Status: Required Changes

**Comment by santiago-rodrig** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required :no_entry_sign:
This project is **almost finished**, but I have to make sure that you are following instructions; because of this I must ask you to do the implement the suggestion I gave to the file `index.html`.

Screen Shot 2020-05-25 at 8 06 42 AM

Notice that the Apple logo is not a link, and that is a requirement.

You can find me in slack as **srodrig**, have a nice day! :wave:

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @KelynPNjeri, congratulations on fixing all the required changes pointed out by the previous TSE's :+1: Our project now meets all the minimum requirements to get approved :slightly_smiling_face: 

You can go ahead and merge to master and submit the project completion form :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**